e2e/testcafe-devextreme/tests/accessibility/chat.ts

```diff
@@ -0,0 +1,69 @@
+import { Message, User, Properties } from 'devextreme/ui/chat.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+import { isMaterial, isMaterialBased } from '../../helpers/themeUtils';
+import { avatarUrl } from '../chat/data';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const getUsers = (length: number, useUrlAvatar: boolean): User[] => {
+  const users = Array.from({ length }, (_, i) => {
+    const user: User = {
+      id: i,
+      name: `User name ${i}`,
+      avatarUrl: useUrlAvatar ? avatarUrl : '',
+    };
+
+    return user;
+  });
+
+  return users;
+};
+
+const getItems = (useUrlAvatar: boolean): Message[] => {
+  const [userFirst, userSecond] = getUsers(2, useUrlAvatar);
+
+  const items: Message[] = [
+    {
+      timestamp: new Date(),
+      text: 'Message text',
+      author: userFirst,
+    },
+    {
+      timestamp: new Date(),
+      text: 'Message text',
+      author: userSecond,
+    },
+  ];
+
+  return items;
+};
+
+const items = [[], getItems(true), getItems(false)];
```

Comment:
Now it's difficult enough for me to compile in a head what will be generated in items. Can we simplify this functional generators?
Particularly, I don't see a reason why we need generate users. Let's just create to constant object `currentUser` and `anotherUser` (with avatarUrl)? It'll be much more clearer for me